test/function_test.py

```diff
@@ -357,6 +356,35 @@ async def test_function_future_async(client, servicer):
         assert future.object_id not in servicer.cleared_function_calls  # keep results around a bit longer for futures
 
 
+def test_function_spawn_exception(client, servicer):
```

Comment:
Kind of scary that we didn't have any tests for spawn or generator calls with exceptions...